client/ayon_core/tools/sceneinventory/model.py

```diff
@@ -129,29 +129,54 @@ def refresh(self, selected=None):
 
         self._clear_items()
 
-        items_by_repre_id = {}
+        items_by_repre_id = collections.defaultdict(list)
+        repre_ids_by_project = collections.defaultdict(set)
         for container_item in container_items:
             # if (
             #     selected is not None
             #     and container_item.item_id not in selected
             # ):
             #     continue
+            project_name = (
+                container_item.project_name or
+                self._controller.get_current_project_name()
+            )
```

Comment:
I though `project_name` is filled with current project on container item creation.
```suggestion
            project_name = container_item.project_name
```